src/cleaning/alru.c

```diff
@@ -466,31 +442,31 @@ int cleaning_policy_alru_initialize(ocf_cache_t cache, int init_metadata)
 {
 	struct ocf_user_part *part;
 	ocf_part_id_t part_id;
-	struct alru_context *alru;
+	struct alru_context *ctx;
 	int error = 0;
 	unsigned i;
 
-	alru = env_vzalloc(sizeof(*alru));
-	if (!alru) {
+	ctx = env_vzalloc(sizeof(*ctx));
```

Comment:
I have a question. Don't we want to remove vmallocs '& Co' from CAS? Just asking because we had this topic long ago.

Reply:
Our biggest concern is using vmallocs in performance sensitive paths (like I/O service). This is a management path, so vmalloc looks like a good idea here. It is slower, but does not require memory pages to be physically continuous like kmalloc. And we don't need physically continuous pages here, since this structure is referenced from CPU only (no DMA / hardware access).

Reply:
Thanks for the explanation.